graphql_compiler/query_comparison.py

```diff
@@ -0,0 +1,79 @@
+from pprint import pformat
+import re
+
+import six
+
+from .debugging_utils import pretty_print_gremlin, pretty_print_match
+from .query_formatting.graphql_formatting import pretty_print_graphql
+
+
+def _get_mismatch_message(expected_blocks, received_blocks):
+    """Create a well-formated error message indicating that two lists of blocks are mismatched."""
+    pretty_expected = pformat(expected_blocks)
+    pretty_received = pformat(received_blocks)
+    return u'{}\n\n!=\n\n{}'.format(pretty_expected, pretty_received)
+
+
+def strip_whitespace(emitted_output):
```

Comment:
I see how `emitted_output` makes sense for backends like match and gremlin, but I don't think it fits with `compare_graphql`, as it's not emitted. My logic is that I've seen functions called `emit_sql` and `emit_match` throughout the code, so I would expect an input to this function to have been outputted from an `emit_*` function. This isn't true for graphql queries.
Perhaps rename to `query_string`? Your call.